sponsors/views.py

```diff
@@ -117,13 +116,20 @@ def form_valid(self, form):
         sponsor = form.save()
 
         uc = use_cases.CreateSponsorshipApplicationUseCase.build()
-        uc.execute(
+        sponsorship = uc.execute(
             self.request.user,
             sponsor,
             benefits_form.get_benefits(),
             benefits_form.get_package(),
         )
+        notified = uc.notifications[1].get_recipient_list(
+            {"user": self.request.user, "sponsorship": sponsorship}
+        )
 
-        response = super().form_valid(form)
+        response = render(
+            self.request,
+            "sponsors/sponsorship_application_finished.html",
+            context={"sponsorship": sponsorship, "notified": notified},
+        )
```

Comment:
Nice refactoring! Definitely redirecting to a success page was me doing some big design up-front

Reply:
yeah, no need for a separate URL, and it's nice to show information back to the user so they build confidence that we "got it"